app/bundles/EmailBundle/Views/Email/list.html.php

```diff
@@ -217,6 +250,9 @@
                         <?php echo $view['content']->getCustomContent('email.stats', $mauticTemplateVars); ?>
                         <?php echo $view['content']->getCustomContent('email.stats.below', $mauticTemplateVars); ?>
                     </td>
+                    <td class="visible-md visible-lg"><?php echo $item->getDateAdded() ? $item->getDateAdded()->format('Y-m-d H:i:s') : ''; ?></td>
+                    <td class="visible-md visible-lg"><?php echo $item->getDateModified() ? $item->getDateModified()->format('Y-m-d H:i:s') : ''; ?></td>
```

Comment:
Use the formats that users can configure like
```suggestion
                    <td class="visible-md visible-lg"><?php echo $item->getDateModified() ? $view['date']->toFull($item->getDateModified()) : ''; ?></td>
```

Reply:
DoneHi @escopecz !

I added the two screenshot.

Removed visible-md is a good idea, i changed it and the method to render the date!